exchanges/gateio/gateio_websocket.go

```diff
@@ -863,14 +852,30 @@ func (g *Gateio) generatePayload(event string, channelsToSubscribe []stream.Chan
 			}
 			params = append(params, intervalString)
 		}
-		payloads[i] = WsInput{
+
+		payload := WsInput{
 			ID:      g.Websocket.Conn.GenerateMessageID(false),
 			Event:   event,
 			Channel: channelsToSubscribe[i].Channel,
 			Payload: params,
 			Auth:    auth,
 			Time:    timestamp.Unix(),
 		}
+
+		if channelsToSubscribe[i].Channel == "spot.book_ticker" {
+			// So to get all assets subscribed it needs to be batched and only
+			// spot.book_ticker can be batched, if not it will take about half
```

Comment:
OBJECTION!!! `spot.tickers` can be batched too https://www.gate.io/docs/developers/apiv4/ws/en/#tickers-channel

Reply:
I was just meaning the orderbooks, but I see your pointerino.